src/pulumi_datarobot_utils/schema/predictions.py

```diff
@@ -145,8 +145,8 @@ class JDBCOutput(Schema):
     createTableIfNotExists: bool = False
     credentialId: str | None = None
     db_schema: str | None = Field(default=None, alias="schema")
-    updateColumns: list[str] | None = Field(default_factory=list)
-    whereColumns: list[str] | None = Field(default_factory=list)
+    updateColumns: list[str] | None = Field(default_factory=lambda: [])
```

Comment:
Pydantic 2.10.2 seems to have updates in typing disallowing using just list 🤔 

<img width="1836" alt="Screenshot 2025-03-26 at 17 16 12" src="https://github.com/user-attachments/assets/57ab0635-16ac-4a47-b032-06e9fff83f34" />